.github/composite_actions/log_metric/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: Log Metric
+description: Log data point to a metric with the provided value. If the metric is not there, it will create one
+# To avoid 'Credentials could not be loaded' calling workflows must include:
```

Comment:
Why is this commented and not removed?

Reply:
This entire comment is meant as a reminder/notice that the calling Github workflow must include the following permissions.  

# To avoid 'Credentials could not be loaded' calling workflows must include:
# permissions:
#  id-token: write
#  contents: read